Makefile

```diff
@@ -35,6 +35,22 @@ CLEANUP_DIR_CMD					 ?= rm -Rf
 # be also cloned from OPENSTACK_REPO/OPENSTACK_BRANCH.
 CHECKOUT_FROM_OPENSTACK_REF	?= false
 
+ADOPTED_SCENARIO					?=false
```

Comment:
names of variables here should not refer to adoption. There's a general case for deploying CP and DP services in different ranges. We should be able to stand up greenfield deployment with different ranges.